trek.js

```diff
@@ -0,0 +1,108 @@
+const URL = 'https://ada-backtrek-api.herokuapp.com/trips';
+
+const reportStatus = (message) => {
+  $('#status-message').html(message);
+};
+
+// LOAD TRIPS
+const loadTrips = () => {
+  const tripsList = $('#tbody');
+  tripsList.empty();
+
+  reportStatus('Loading trips! Please wait...');
+  $('#table').show();
```

Comment:
same comment as below, but: the ID `#table` isn't a very helpful ID name! Which table is it?